app/helpers/maintenance_tasks/tasks_helper.rb

```diff
@@ -100,5 +100,25 @@ def csv_file_download_path(run)
         only_path: true
       )
     end
+
+    # Return the appropriate field tag for the parameter
+    def parameter_field(form_builder, parameter_name)
+      # rubocop:disable Layout/LineLength
+      case form_builder.object.class.attribute_types[parameter_name]
```

Comment:
Unfortunately `attribute_types` is not documented, and I'm not sure there's another way to get to the types.

Reply:
Checked in with our `#rails` channel and [Rafael mentioned](https://shopify.slack.com/archives/C08CPK9JB/p1638466958389600?thread_ts=1638461557.389100&cid=C08CPK9JB) that `attribute_types` should be fine to use since we're using `ActiveModel::Attributes`